src/main/java/org/folio/dew/service/FolioTenantService.java

```diff
@@ -0,0 +1,63 @@
+package org.folio.dew.service;
+
+import java.sql.ResultSet;
+
+import lombok.extern.log4j.Log4j2;
+import org.apache.commons.lang3.BooleanUtils;
+import org.springframework.context.annotation.Primary;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+
+import org.folio.dew.config.kafka.KafkaService;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.liquibase.FolioSpringLiquibase;
+import org.folio.spring.service.TenantService;
+import org.folio.tenant.domain.dto.TenantAttributes;
+
+@Log4j2
```

Comment:
Logger is never used in the class

Reply:
45 line: `log.error(e.getMessage(), e);`

Reply:
Sorry @Shans-Kaluhin 😄